clients/apps/web/src/components/CommandPalette/commands/trigger.tsx

```diff
@@ -23,7 +23,8 @@ export const useCommandPaletteTrigger = (onTrigger: () => void): void => {
   useEffect(() => {
     const handleKeyPress = (e: KeyboardEvent) => {
       const controlKeyPressed = isMac(navigator) ? e.metaKey : e.ctrlKey
-      if (e.key === KEY && controlKeyPressed) {
+      if (e.key.toLowerCase() === KEY && controlKeyPressed) {
```

Comment:
Sorry, I don't get it 😅 How lowering the case would solve the issue? 😅

Reply:
Actually the solution is preventing the default behaviour of the browser as when we click ctrl+k, the browser catches it first so I added `e.preventDefault` to prevent the default behaviour of the browser.

Reply:
added lower casing just for the sake of a double check for the command.

Reply:
Oh yeah, I see, sorry. Let's merge it — can't do any harm either way — and I'll check on my Windows computer then when I have a minute